sympy/stats/tests/test_rv.py

```diff
@@ -214,12 +214,11 @@ def test_Sample():
     if not numpy:
         skip('Numpy is not installed. Abort tests')
     #Test Issue #21563: Output of sample must be a float or array
-    assert isinstance(sample(X), numpy.int64)
+    assert isinstance(sample(X), (numpy.int32, numpy.int64))
```

Comment:
Why is this changed?

Reply:
For some reason it returned `numpy.int32` on my computer. Not sure why though. I do not think it relates to the changes I've made, so I guess that it may return that on 32-bit machines/numpy/python (which I do not think I have...).

Reply:
And any integer type should be OK considering that there are 6 options...

Reply:
I can just confirm that I get this on my current master as well. Must have installed a 32-bit version by mistake?

Anyway, I think it makes sense that this passes even if you have a 32-bit version locally. (Noted that I was running pywin32 although I did install the 64-bit miniconda as far as I (and my web history) can tell...)

Reply:
Yes, I think this is fine.